cloudstack/resource_cloudstack_instance.go

```diff
@@ -144,7 +144,12 @@ func resourceCloudStackInstance() *schema.Resource {
 				Optional: true,
 			},
 
-			"hostid": {
+			"host_id": {
```

Comment:
this can impact existing users using hostid param, doc/release notes update needed. cc @kiranchavala

Reply:
@sureshanaparti, I think existing users are not affected as the hostid param was not present in 0.4.0 release. 

hostid fix is also  planned for 0.5.0 release, I just changed it from hostid to host_id to follow the convention

https://github.com/apache/cloudstack-terraform-provider/pull/52

Reply:
ok @kiranchavala